src/helm/benchmark/scenarios/vision_language/utils_latex.py

```diff
@@ -0,0 +1,85 @@
+from typing import Optional, Tuple
+
+import io
+import os
+
+from helm.common.optional_dependencies import OptionalDependencyNotInstalled
+
+try:
+    from latex import build_pdf
+    from pdf2image import convert_from_bytes
+    from PIL import ImageOps
+except ModuleNotFoundError as e:
+    raise OptionalDependencyNotInstalled(
+        f"Optional dependency {e.name} is not installed. "
+        "Please run `pip install latex pdf2image Pillow` to install it."
```

Comment:
Add this to the vlm optional dependency group in `setup.cfg`?

Reply:
Done